src/frontend/utils/tests/factories.ts

```diff
@@ -155,6 +158,15 @@ export const videoMockFactory = (video: Partial<Video> = {}): Video => {
   };
 };
 
+export const liveMockFactory = (live: Partial<Live> = {}): Live => {
+  return {
+    ...videoMockFactory(live),
+    live_state: liveState.IDLE,
+    live_type: LiveModeType.JITSI,
```

Comment:
Should this be a live or a LiveJitsi ? (https://github.com/openfun/marsha/blob/ef98aeea917e1620a052779d8293b64752f204e1/src/frontend/types/tracks.ts#L230)

Reply:
For me it's a live, predefined as a jitsi one :p
IMO, it would be strange to see
```
const rawLive = liveJitsiMockFactory({live_type: LiveModeType.RAW})
```